public/js/selfoss-base.js

```diff
@@ -97,6 +118,75 @@ var selfoss = {
         
         return values;
     },
+
+
+    loggedin: false,
+
+
+    setSession: function() {
+        Cookies.set('onlineSession', 'true', {expires: 10,
+                                              path: window.location.pathname});
+        selfoss.loggedin = true;
+    },
+
+
+    clearSession: function() {
+        Cookies.remove('onlineSession', {path: window.location.pathname});
+        selfoss.loggedin = false;
+    },
+
+
+    hasSession: function() {
+        selfoss.loggedin = Cookies.get('onlineSession') == 'true';
```

Comment:
This cookie is a hint that there was an online session when last loaded, and will be removed at startup when PHPSESSID is cleared. When that PHPSESSID is cleared, you get like before the 403 error after each ajax request. The next step is to fix #919 .